cli.sh

```diff
@@ -44,17 +44,34 @@ if [ $ACTIONS == $START ]; then
 
 elif [ "$ACTIONS" == "$DEPLOY" ]; then
     LOGIN_IN_VM="SSH into a Virtual Machine"
-    DEPLOY_SPA="Deploy SPA on a Virtual Machine"
+    SET_VM_ENV="Set up Virtual Machine for Deployment"
 
-    ACTIONS=$(gum choose --cursor-prefix "[ ] " --selected-prefix "[✓] " --no-limit "$LOGIN_IN_VM" "$DEPLOY_SPA")
+    ACTIONS=$(gum choose --cursor-prefix "[ ] " --no-limit "$LOGIN_IN_VM" "$SET_VM_ENV")
```

Comment:
Why did you remove the selected prefix?

Reply:
I supposed I had to remove it. Did you mean I should've just changed the icon (a.k.a the tick mark)?